docs/content/users/providers/platform.md

```diff
@@ -6,30 +6,40 @@ ddev provides integration with the [Platform.sh Website Management Platform](htt
 
 ddev's Platform.sh integration pulls database and files from an existing Platform.sh site/environment into your local system so you can develop locally.
 
-## Platform.sh Quickstart
+## Platform.sh Global Configuration
+
+You need to obtain and configure an API token first. This is only needed once.
+
+1. Login to the Platform.sh Dashboard and go to Account->API Tokens to create an API token for ddev to use.
+2. Add the API token to the `web_environment` section in your global ddev configuration at ~/.ddev/global_config.yaml:
+
+```yaml
+web_environment:
+  - PLATFORMSH_CLI_TOKEN=abcdeyourtoken
+```
+
+## Platform.sh per Project Configuration
 
 1. Check out the site from platform.sh and then configure it with `ddev config`. You'll want to use `ddev start` and make sure the basic functionality is working.
-2. Obtain and configure an API token.
-   a. Login to the Platform.sh Dashboard and go to Account->API Tokens to create an API token for ddev to use.
-   b. Add the API token to the `web_environment` section in your global ddev configuration at ~/.ddev/global_config.yaml:
+2. Add PLATFORM_PROJECT and PLATFORM_ENVIRONMENT variables to your project.
 
-   ```yaml
-   web_environment:
-   - PLATFORMSH_CLI_TOKEN=abcdeyourtoken
-   ```
+    * Either in `.ddev/config.yaml` or a `.ddev/config.*.yaml` file:
 
-3. Add PLATFORM_PROJECT and PLATFORM_ENVIRONMENT variables to your project `.ddev/config.yaml` or a `.ddev/config.*.yaml`:
+        ```yaml
+        web_environment:
+          - PLATFORM_PROJECT=nf4amudfn23biyourproject
+          - PLATFORM_ENVIRONMENT=main
+        ```
 
-```yaml
-   web_environment:
-   - PLATFORM_PROJECT=nf4amudfn23biyourproject
-   - PLATFORM_ENVIRONMENT=main
- ```
-
-You can also do this with `ddev config --web-environment-add="PLATFORM_PROJECT=nf4amudfn23bi,PLATFORM_ENVIRONMENT=main"`
-4. `ddev restart`
-5. Run `ddev pull platform`. After you agree to the prompt, the current upstream database and files will be downloaded.
-6. Optionally use `ddev push platform` to push local files and database to platform.sh. Note that `ddev push` is a command that can potentially damage your production site, so this is not recommended.
+    * Or with a command from your terminal:
+
+        ```bash
```

Comment:
This isn't working out quite right, it shows as `bash ddev config` - above I see you did a newline below the ```yaml, maybe that will solve it. Ir just use ``` and not ````bash

Thanks!
<img width="960" alt="Platform_sh_Integration_-_DDEV_Docs" src="https://user-images.githubusercontent.com/112444/184353034-ff945747-1b56-4a26-9241-42bca454db05.png">

Reply:
There was one extra whitespace. I removed it

Thanks